Adding 1.8 install  changes (#5002)
fishtown-analytics/docs.getdbt.com: website/docs/docs/core/pip-install.md

```

### Installing the adapter

> Once you decide [which adapter](/docs/supported-data-platforms) you're using, you can install using the command line. Beginning in v1.8, you must install both dbt Core and the desired adapter because dbt-core has been decoupled from the adapter installation.

Author: ```suggestion
Once you decide [which adapter](/docs/supported-data-platforms) you're using, you can install using the command line. Beginning in v1.8, you must install both dbt Core and the desired adapter because `dbt-core` has been decoupled from the adapter installation.
```

> Once you decide [which adapter](/docs/supported-data-platforms) you're using, you can install using the command line. Beginning in v1.8, you must install both dbt Core and the desired adapter because `dbt-core` has been decoupled from the adapter installation.

Author: ```suggestion
Once you decide [which adapter](/docs/supported-data-platforms) you're using, you can install using the command line. Beginning in v1.8, installing an adapter no longer automatically installs `dbt-core`. This is because adapters and dbt Core versions have been decoupled from each other so we no longer want to overwrite existing dbt-core installations.
```

> Once you decide [which adapter](/docs/supported-data-platforms) you're using, you can install using the command line. Beginning in v1.8, installing an adapter no longer automatically installs `dbt-core`. This is because adapters and dbt Core versions have been decoupled from each other so we no longer want to overwrite existing dbt-core installations.

Reviewer: love this! much clearer 
```suggestion
Once you decide [which adapter](/docs/supported-data-platforms) you're using, you can install using the command line. Beginning in v1.8, installing an adapter does not automatically install `dbt-core`. This is because adapters and dbt Core versions have been decoupled from each other so we no longer want to overwrite existing dbt-core installations.
```

<VersionBlock firstVersion="1.8">

```shell

> python3 -m pip install dbt-core dbt-ADAPTER_NAME. 

Author: @dbeatty10 or @colin-rogers-dbt I changed this to say `python3` is this right? I couldn't get `python` to work but perhaps this is my local set up? I noticed the requirements state Python 3+.

Reviewer 2: It's reasonable to either leave it as `python` or update it to `python3`. Either way, the instructions are bound to not work for someone though 😢

If we update it to `python3`, we should update all the same places as in https://github.com/dbt-labs/docs.getdbt.com/pull/4498.

There's some more background on the differences between these names here:
- https://www.reddit.com/r/learnpython/comments/mf7t0n/why_python3_in_command_prompt_vs_python/
- https://stackoverflow.com/questions/64801225/python-or-python3-what-is-the-difference

Python 2 reached end of life on January 1st, 2020, and so it's been reasonable to use just `python` rather than `python3` since then.

Reviewer 3: +1 we should use `python`, while mac defaults to python2 (perhaps what's happened on your computer) python3 is the default so no need to specify

Author: OK happy to change it! Thanks for your input.

Author: Changed in https://github.com/dbt-labs/docs.getdbt.com/pull/5002/commits/bbe77e00a1a6c9d395fe3c531612f0ccabd9772f

> python3 -m pip install dbt-core dbt-ADAPTER_NAME. 

Author: ```suggestion
python -m pip install dbt-core dbt-ADAPTER_NAME. 
```

> python -m pip install dbt-core dbt-ADAPTER_NAME. 

Reviewer: ```suggestion
python -m pip install dbt-core dbt-ADAPTER_NAME
```

```

</VersionBlock>

<VersionBlock lastVersion="1.7">

```shell

> python3 -m pip install dbt-ADAPTER_NAME

Author: ```suggestion
python -m pip install dbt-ADAPTER_NAME
```

```

</VersionBlock>

For example, if using Postgres:

<VersionBlock lastVersion="1.8">

```shell

> python3 -m pip install dbt-core dbt-postgres

Author: ```suggestion
python -m pip install dbt-core dbt-postgres
```

> <VersionBlock lastVersion="1.8">

Reviewer: ```suggestion
<VersionBlock firstVersion="1.8">
```

```

All adapters build on top of `dbt-core`. Some also depend on other adapters: for example, `dbt-redshift` builds on top of `dbt-postgres`. In that case, you would see those adapters included by your specific installation, too.
</VersionBlock>

<VersionBlock lastVersion="1.7">

```shell

> python3 -m pip install dbt-postgres

Author: ```suggestion
python -m pip install dbt-postgres
```

```

This will install `dbt-core` and `dbt-postgres` _only_:

```shell
$ dbt --version
installed version: 1.0.0
   latest version: 1.0.0

Up to date!

Plugins:
  - postgres: 1.0.0
```

Some adapters depend on other adapters. For example, `dbt-redshift` builds on top of `dbt-postgres`. In that case, you would see those adapters included by your specific installation, too.
</VersionBlock>

### Upgrade adapters

To upgrade a specific adapter plugin:

```shell

> python3 -m pip install --upgrade dbt-<adapter>

Author: ```suggestion
python -m pip install --upgrade dbt-<adapter>
```

```

### Install dbt-core only

If you're building a tool that integrates with dbt Core, you may want to install the core library alone, without a database adapter. Note that you won't be able to use dbt as a CLI tool.

```shell

> python3 -m pip install dbt-core

Author: ```suggestion
python -m pip install dbt-core
```

```

### Change dbt Core versions

You can upgrade or downgrade versions of dbt Core by using the `--upgrade` option on the command line (CLI). For more information, see [Best practices for upgrading in Core versions](/docs/dbt-versions/core#best-practices-for-upgrading).

To upgrade dbt to the latest version:

```

> python3 -m pip install --upgrade dbt-core

Author: ```suggestion
python -m pip install --upgrade dbt-core
```

```

To downgrade to an older version, specify the version you want to use. This command can be useful when you're resolving package dependencies. As an example:

```

> python3 -m pip install --upgrade dbt-core==0.19.0

Author: ```suggestion
python -m pip install --upgrade dbt-core==0.19.0
```

If you have workflows or integrations that relied on installing the package named `dbt`, you can achieve the same behavior going forward by installing the same five packages that it used:

```shell

> python3 -m pip install \

Author: ```suggestion
python -m pip install \
```
